src/common/aes/aes.h

```diff
@@ -54,25 +54,48 @@ void OQS_AES128_ECB_enc_sch(const uint8_t *plaintext, const size_t plaintext_len
  * Function to fill a key schedule given an initial key for use in ECB mode encryption.
  *
  * @param key            Initial Key.
- * @param schedule       Abstract data structure for a key schedule.
+ * @param ctx            Abstract data structure for a key schedule.
  */
-void OQS_AES256_ECB_load_schedule(const uint8_t *key, void **schedule);
+void OQS_AES256_ECB_load_schedule(const uint8_t *key, void **ctx);
 
 /**
- * Function to fill a key schedule given an initial key for use in CTR mode.
+ * Function to initialize a context and fill a key schedule given an initial key for
+ * use in CTR mode.
  *
  * @param key            Initial Key.
- * @param schedule       Abstract data structure for a key schedule.
+ * @param ctx            Abstract data structure for a key schedule.
  */
-void OQS_AES256_CTR_load_schedule(const uint8_t *key, void **schedule);
+void OQS_AES256_CTR_inc_init(const uint8_t *key, void **ctx);
+
+/**
+ * Function to fill a context given an IV for use in CTR mode.
+ *
+ * Handles a 12- or 16-byte IV.  If a 12-byte IV is given, then 4 counter
+ * bytes are initialized to all zeros.
+ *
+ * @param iv             Initialization Vector.
+ * @param iv_len         Length of the initialization vector.
+ * @param ctx            Abstract data structure for IV.
+ */
+void OQS_AES256_CTR_inc_iv(const uint8_t *iv, size_t iv_len, void *ctx);
+
+/**
+ * Function to fill a context fiven an IV for use in CTR mode.
```

Comment:
typo "fiven". In general the question: What does "inc" stand for in the function name?

Reply:
typo fixed in 0b91245833c1bb9756519316fb13b8cb4d67d2cd